src/runtime_src/core/include/deprecated/xrt.h

```diff
@@ -80,6 +80,34 @@ typedef unsigned int xclBufferHandle;
 #endif
 #define XRT_NULL_BO NULLBO
 
+/*
+ * typedef xrt_buffer_handle - opaque buffer handle
+ *
+ * A buffer handle of xrt_buffer_handle kind is obtained by allocating buffer
+ * objects through ISHIM API. The buffer handle is used by XRT ISHIM APIs
+ * that operate on on buffer objects.
+ */
+typedef void * xrt_buffer_handle;
+#define XRT_INVALID_BUFFER_HANDLE	NULL
```

Comment:
Hmm...would `nullptr `be better here. 

Note: If the code is used in C code then the answer would be no :^).

Reply:
The answer is no for exactly the reason you mention ;-)